R/tab_create_modify.R

```diff
@@ -1036,12 +1036,33 @@ tab_style <- function(data,
   # Perform input object validation
   stop_if_not_gt(data = data)
 
-  # Intercept font styles that require registration
-  if ("cell_text" %in% names(style)) {
+  # Upgrade `style` to be within a list if not provided as such
+  if (!inherits(style, "list")) {
```

Comment:
```suggestion
  if (inherits(style, "cell_styles")) {
```
The `!inherits(style, "list")` was confusing to me because `cell_styles` pass `is.list()`

Reply:
Now done. Thanks!